pom.xml

```diff
@@ -49,7 +49,9 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
 
-        <asciidoctorj.version>1.5.3.2</asciidoctorj.version>
+        <asciidoctorj.version>1.6.0-alpha.3</asciidoctorj.version>
```

Comment:
this solves issues with generating PDF (see https://github.com/hibernate/hibernate-validator/pull/729). So I'd say that we have next options:

- release two versions of extensions: one based on 1.5.5 another on 1.6 and use them in BV and HV corresponding.
- not use extension for HV (at least for now) and revert to 1.5.5. 
- wrap html generation of spec for BV with maven and then run something like `mvn clean package` from ant to only generate html version of spec leaving ant for everything else.